scripts/llogcolor

```diff
@@ -30,58 +30,72 @@ import subprocess
 import signal
 import errno
 
-colors =  { 'default'       :    "\033[0m",
-            'bold'          :    "\033[1m",
-            'underline'     :    "\033[4m",
-            'blink'         :    "\033[5m",
-            'reverse'       :    "\033[7m",
-            'concealed'     :    "\033[8m",
-
-            'black'         :    "\033[30m", 
-            'red'           :    "\033[31m",
-            'green'         :    "\033[32m",
-            'yellow'        :    "\033[33m",
-            'blue'          :    "\033[34m",
-            'magenta'       :    "\033[35m",
-            'cyan'          :    "\033[36m",
-            'white'         :    "\033[37m",
-
-            'bright_black'  :    "\033[90m", 
-            'bright_red'    :    "\033[91m",
-            'bright_green'  :    "\033[92m",
-            'bright_yellow' :    "\033[93m",
-            'bright_blue'   :    "\033[94m",
-            'bright_magenta':    "\033[95m",
-            'bright_cyan'   :    "\033[96m",
-            'bright_white'  :    "\033[97m",
-
-            'on_black'      :    "\033[40m", 
-            'on_red'        :    "\033[41m",
-            'on_green'      :    "\033[42m",
-            'on_yellow'     :    "\033[43m",
-            'on_blue'       :    "\033[44m",
-            'on_magenta'    :    "\033[45m",
-            'on_cyan'       :    "\033[46m",
-            'on_white'      :    "\033[47m" }
+colors = {
```

Comment:
The values of `color` could be aligned if anyone cares. The python style guide (PEP 8) says not to for assignment statements, and I don't see anything for dict key-value pairs.